RPS.html

```diff
@@ -84,13 +84,10 @@
       textResultGame.forEach((element) => resultGameDiv.appendChild(element));
     }
 
-    const rockBtn = document.querySelector('#rock');
-    const paperBtn = document.querySelector('#paper');
-    const scissorsBtn = document.querySelector('#scissors');
-
-    rockBtn.addEventListener('click', () => game(rock));
-    paperBtn.addEventListener('click', () => game(paper));
-    scissorsBtn.addEventListener('click', () => game(scissors));
+    gameOptions.forEach((element) => {
+      const button = document.querySelector('#' + element);
+      button.addEventListener('click', () => game(element));
+    });
   </script>
  </body>
 </html>
```

Comment:
Remember to add a blank line at the end of every file

Reply:
It will be done